src/assets/theme/index.tsx

```diff
@@ -1,8 +1,56 @@
 import { createMuiTheme } from '@material-ui/core';
-
+import { css, FlattenSimpleInterpolation } from 'styled-components';
 import palette from './palette';
 import typography from './typography';
 import overrides from './overrides';
+import { COLOR_MAP } from 'common/colors';
+
+interface ThemeColors {
+  green: string;
+  greenDark: string;
+  lightBlue: string;
+}
+
+const colors: ThemeColors = {
+  green: COLOR_MAP.GREEN.BASE,
+  greenDark: COLOR_MAP.GREEN.DARK,
+  lightBlue: COLOR_MAP.BLUE,
+};
+
+interface ThemeFonts {
+  subtitle1: FlattenSimpleInterpolation;
+  disclaimer: FlattenSimpleInterpolation;
+  link: FlattenSimpleInterpolation;
+}
+
+/**
+ * High-level typography groups
+ */
+const fonts: ThemeFonts = {
```

Comment:
im confused about this but it might be better discussed on a call tomorrow. i thought the original goal we landed on was to have a set of fonts that are identified by their styling (ie. `monospaceBold`, `regularBold`, `regularItalic` -- regular being our main font ie. Roboto), rather than defining them by their use case (`subtitle1`, `disclaimer`). starting from the point of defining them by their use case feels limiting/not all that flexible

Reply:
we probably need to do both, right now there are more high-level elements repeated (headings, disclaimers, etc) than low-level elements (regularBold)